packages/lesswrong/lib/abTests.ts

```diff
@@ -250,7 +250,7 @@ export const postPageFixedDigestAd = new ABTest({
 
 export const jobAdDescription = new ABTest({
   name: "jobAdDescription",
-  active: true,
+  active: false,
```

Comment:
I think it's ok to delete an A/B test after I'm done with it? But I'm not totally sure so for now I will just mark it as inactive

Reply:
It should be fine to delete imo